app/views/clients/_form.html.slim

```diff
@@ -1,23 +1,19 @@
 = simple_form_for(@client) do |f|
   = simple_error_notice f
-  = f.input :salutation, as: :select, collection: Client.salutation_collection,
-              include_blank: true
+  = f.input :salutation, as: :select, collection: Client::SALUTATIONS, include_blank: true
 
   .row
     = f.simple_fields_for :contact do |c|
-      .col-xs-12.col-md-6
-        = c.input :first_name, required: true
-      .col-xs-12.col-md-6
-        = c.input :last_name, required: true, label: t_attr(:last_name)
-    .col-xs-12.col-md-6
-      = f.input :birth_year, as: :select, collection: Client.year_collection, include_blank: true
-    .col-xs-12.col-md-6
-      = f.input :nationality, as: :country
+      .col-xs-12.col-md-6= c.input :first_name, required: true
+      .col-xs-12.col-md-6= c.input :last_name, required: true, label: t_attr(:last_name)
+    .col-xs-12.col-md-4= f.input :birth_year, as: :select, collection: Client.year_collection,
+                                 include_blank: true
+    .col-xs-12.col-md-4= f.input :nationality, as: :country
+    .col-xs-12.col-md-4= f.input :entry_date
 
   = f.simple_fields_for :contact do |c|
     = render 'contacts/fields', f: c
 
-
   fieldset
     legend= t_attr(:permit, Client)
     = f.input :permit, collection: Client::PERMITS, as: :radio_buttons, label: false
```

Comment:
I just meant to use `horizontal_radio_and_checkboxes` instead of the vertical ones. I think it should stay next to the nationality since it was requested back then. sorry for the misunderstanding..

Reply:
adding `wrapper: :horizontal_radio_and_checkboxes` and unless it is part of the requested feedback, maybe it could stay as it is?

Reply:
it was initially requested back then, when permit was implemented and I had a call with Zaida yesterday and she confirmed, that these 3 information should stay together

Reply:
Do they have preference to radio buttons over dropdowns? Because I haven't yet found how to override the default settings and make the radio buttons horizontal.

Reply:
yes they requested radio buttons .. :/

Reply:
Have they also requested them horizontal?

Reply:
Ok, didn't realize it was optional. (It should definitely be possible, I just haven't found the way to do it.) So unless there aren't any other requested changes, it should be ready. :)